Utils/Accessibility.swift

```diff
@@ -0,0 +1,40 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import UIKit
+
+public protocol AccessibilityActionsSource: class {
+    func accessibilityCustomActionsForView(view: UIView) -> [UIAccessibilityCustomAction]?
+}
+
+public class Action {
```

Comment:
That is not the way I am thinking about it. `Action` really represents a generic action that has in principle nothing to do with accessibility, and from which both `UIAccessibilityCustomAction` and `UIAlertAction` can be generated.